### PR TITLE
New: added production template for jaerger with elasticsearch

### PR DESCRIPTION
**Pull request by Dogild**
Signed-off-by: Alexandre Wilhelm <contact.dogild@gmail.com>

### CODE REVIEW

File: production-elasticsearch/jaeger-production-template-with-elasticsearch.yml [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+          - containerPort: 14268
+            protocol: TCP
+          - containerPort: 9411
+            protocol: TCP
+          command:
+            - "/go/bin/collector-linux"
+            - "--span-storage.type=elasticsearch"
+            - "--es.server-urls=http://elasticsearch:9200"
+            - "--es.username=elastic"
+            - "--es.password=changeme"
```
Comment (jpkrohling) [inline production-elasticsearch/jaeger-production-template-with-elasticsearch.yml]: Will this work out of the box? Or should a user actually change it before using?

File: pom.xml [hunk 2]
```diff
@@ -50,6 +50,7 @@
     <module>jaegertracing-kubernetes-deployment-itest</module>
     <module>all-in-one</module>
     <module>production</module>
```
Comment (jpkrohling) [inline pom.xml]: Could you open an issue, to rename the existing one to `production-cassandra`?

File: production-elasticsearch/jaeger-production-template-with-elasticsearch.yml [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+          name: jaeger-query
+          ports:
+          - containerPort: 16686
+            protocol: TCP
+          command:
+          - "/go/bin/query-linux"
+          - "--span-storage.type=elasticsearch"
+          - "--es.server-urls=http://elasticsearch:9200"
+          - "--es.username=elastic"
+          - "--es.password=changeme"
```
Comment (jpkrohling) [inline production-elasticsearch/jaeger-production-template-with-elasticsearch.yml]: Same as above

File: production-elasticsearch/jaeger-production-template-with-elasticsearch.yml (hunk 1 above)
Comment (pavolloffay) [reply production-elasticsearch/jaeger-production-template-with-elasticsearch.yml]: IIRC this is the default user/pass https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#_security_note. 

Or we can disable auth altogether.

**Comment by pavolloffay** [comment]
Tests look fine https://travis-ci.org/jaegertracing/jaeger-kubernetes/builds/282368267#L1519

**Pull request merged**